src/components/daydream-controls.js

```diff
@@ -18,14 +16,16 @@ var GAMEPAD_ID_PREFIX = 'Daydream Controller';
  */
 module.exports.Component = registerComponent('daydream-controls', {
   schema: {
-    hand: {default: DEFAULT_HANDEDNESS}, // This informs the degenerate arm model.
+    hand: {default: ''}, // This informs the degenerate arm model.
```

Comment:
(i.e. just because there is only one controller doesn't mean apps have to use hand-agnostic, symmetrical models, because the system is designed to tell you which hand that one controller represents, and we should reflect that when requested)